quicklogic/openfpga/tests/quicklogic_testsuite/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+add_custom_target(all_openfpga_k4_N8_tests_no_adder)
```

Comment:
These should all be under `third_party`.

Reply:
After presenting this to the QuickLogic people I removed the test suite. It is to be added as a submodule here or to the symbiflow-examples repo.